### ISSUE TITLE
Move Repository to Examples

### ISSUE THREAD

**Issue by MuellerSeb** [GeoStat-Framework/ogs5py_benchmarks#1]
This repository could / should be moved to the new [GeoStat-Examples](https://github.com/GeoStat-Examples) organization.

@fhesze , @AlrauneZ , @LSchueler : You agree?

**Comment by LSchueler** [comment]
Yeah, as long as the word "benchmark" stays prominently visible, I dig it.
